rest-api-spec/src/main/resources/rest-api-spec/test/indices.create/10_basic.yaml

```diff
@@ -31,6 +31,35 @@
   - match: { test_index.settings.index.number_of_replicas: "0"}
 
 ---
+"Create index with too large wait_for_active_shards":
+
+  - do:
+      indices.create:
+        index: test_index
+        timeout: 100ms
+        master_timeout: 100ms
+        wait_for_active_shards: 6
+        body:
+          settings:
+            number_of_replicas: 5
+
+  - match: { shards_acknowledged: false }
```

Comment:
same reason, the low timeout doesn't necessarily guarantee index creation completed in time, and i didn't want the tests failing because of that